### PR TITLE
Fixing consumer authentication documentation

### PR DESCRIPTION
**Pull request by lpbarber**
Fixing consumer authentication documentation

### CODE REVIEW

File: pages/Documents/Security&Authentication/ConsumerAuthentication/implementation-guide.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-pagename: Detailed API
+pagename: Implementation Guide
 redirect_from:
```
Comment (j9t) [inline pages/Documents/Security&Authentication/ConsumerAuthentication/implementation-guide.md:3]: Please add a redirect from the old file name (consumer-authentication-detailed-api.html).

**Review by j9t — CHANGES_REQUESTED** [review]
Makes thing clearer—thanks!

File: pages/Documents/Security&Authentication/ConsumerAuthentication/implementation-guide.md (hunk 1 above)
Comment (j9t) [reply pages/Documents/Security&Authentication/ConsumerAuthentication/implementation-guide.md:3]: @lpbarber, if you could make this change we can roll out the update.

**Comment by lpbarber** [comment]
Done.

Yaniv Barber
Prin, Tech Prod Mgt

<https://pr.liveperson.com/2022-03-08-Fast-Company-names-LivePerson-the-1-Most-Innovative-AI-Company-in-the-World?utm_medium=email&utm_source=gmail&utm_campaign=signature&utm_content=2022FastCompany>


On Wed, Oct 26, 2022 at 10:40 AM Jens Oliver Meiert <
***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In
> pages/Documents/Security&Authentication/ConsumerAuthentication/implementation-guide.md
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LivePersonInc_developer-2Dcenter_pull_1982-23discussion-5Fr1005327100&d=DwMCaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=9DFymaCbg0Bg44CiynVySrTs7llgcwF01--0jqrc9o4&m=Iwd1NWzk266Z_smgOr9Gz9Y-XQqrV-BQY-KybJbzhkXICZ3JfNCEV5f6qfZJZ-s2&s=4kj0Y9KPlGDBRLaGwABF7Ffubl4hOYRzBsPHOzQiLLY&e=>
> :
>
> > @@ -1,17 +1,17 @@
>  ---
> -pagename: Detailed API
> +pagename: Implementation Guide
>  redirect_from:
>
> @lpbarber
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_lpbarber&d=DwMCaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=9DFymaCbg0Bg44CiynVySrTs7llgcwF01--0jqrc9o4&m=Iwd1NWzk266Z_smgOr9Gz9Y-XQqrV-BQY-KybJbzhkXICZ3JfNCEV5f6qfZJZ-s2&s=KjhM_dEylwCPA-dNPUEuDUwo1oymH2CVVgkeN3GygmI&e=>,
> if you could make this change we can roll out the update.
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_LivePersonInc_developer-2Dcenter_pull_1982-23discussion-5Fr1005327100&d=DwMCaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=9DFymaCbg0Bg44CiynVySrTs7llgcwF01--0jqrc9o4&m=Iwd1NWzk266Z_smgOr9Gz9Y-XQqrV-BQY-KybJbzhkXICZ3JfNCEV5f6qfZJZ-s2&s=4kj0Y9KPlGDBRLaGwABF7Ffubl4hOYRzBsPHOzQiLLY&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ADCBTIX7D4GKZ7JZOROJYUTWFDN7NANCNFSM6AAAAAARKF2ERQ&d=DwMCaQ&c=NxS7LVD4EucgUR9_G6bWzuqhmQ0xEJ2AZdqjz4WaSHU&r=9DFymaCbg0Bg44CiynVySrTs7llgcwF01--0jqrc9o4&m=Iwd1NWzk266Z_smgOr9Gz9Y-XQqrV-BQY-KybJbzhkXICZ3JfNCEV5f6qfZJZ-s2&s=yvjLD0TwG0LRGJa2gLo_hOkzXAC57INmPVF7TBP0pyE&e=>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
This message may contain confidential and/or privileged information. 
If 
you are not the addressee or authorized to receive this on behalf of the 
addressee you must not use, copy, disclose or take action based on this 
message or any information herein. 
If you have received this message in 
error, please advise the sender immediately by reply email and delete this 
message. Thank you.

**Review by j9t — APPROVED** [review]

**Pull request merged**